analysers/analyser_merge_poi_FR.py

```diff
@@ -0,0 +1,320 @@
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'catholic'},
+        ), dict(
+            #  15289
+            item = 8100,
+            id = 10002,
+            level = 3,
+            title = T_('Place of worship not integrated'),
+            select = {'nature': 'Culte chrétien', 'nature_detaillee': ''},
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'amenity': 'place_of_worship', 'religion': 'christian'},
+        ),
+        dict(
+            #   1417 Oratoire
+            item = 8100,
+            id = 10003,
+            level = 3,
+            title = T_('Place of worship not integrated'),
+            select = {'nature': 'Culte chrétien', 'nature_detaillee': 'Oratoire'},
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'historic': 'wayside shrine', 'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'catholic'},
+        ),
+        dict(
+            #   2255 Temple protestant
+            item = 8100,
+            id = 10004,
+            level = 3,
+            title = T_('Place of worship not integrated'),
+            select = {'nature': 'Culte chrétien', 'nature_detaillee': 'Temple protestant'},
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'protestant'},
+        ),
+        dict(
+            #     64 Eglise orthodoxe
+            item = 8100,
+            id = 10005,
+            level = 3,
+            title = T_('Place of worship not integrated'),
+            select = {'nature': 'Culte chrétien', 'nature_detaillee': 'Eglise orthodoxe'},
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'orthodox'},
+        ),
+            #     46 Enclos paroissial
+            #     42 Eglise simultanée
+        # 54455 Enseignement primaire
+        # 53718 Espace public
+        # 38865 Mairie
+        dict(
+            item = 8110,
+            id = 10100,
+            level = 3,
+            title = T_('Townhall not integrated'),
+            select = {'nature': 'Mairie', 'nature_detaillee': ['', 'Mairie de commune associée ou déléguée', 'Mairie annexe', 'Hôtel de ville', 'Mairie d\'arrondissement']},
+            tags = {'amenity': 'townhall'},
+        ),
+        # 27275 Monument
+        dict(
+            #  13600 Monument aux morts
+            item = 8010,
+            id = 10200,
+            level = 3,
+            title = T_('Memorial not integrated'),
+            select = {'nature': 'Monument', 'nature_detaillee': 'Monument aux morts'},
+            tags = {'historic': 'memorial'},
+        ),
+            #   7111
+            #   5082 Stèle
+            #   1304 Statue
+            #    188 Mémorial
```

Comment:
Pour tous les nature: Monument (pour ceux sans nature_detaillee je ne sais pas si ça convient...) :
historic=memorial https://wiki.openstreetmap.org/wiki/FR:Tag:historic%3Dmemorial
avec le bon memorial=*
Sur la page du wiki, il y a pour memorial=* : war_memorial, stele, statue

Reply:
Low volume. I prefer skip this one.